### PR TITLE
BGDIINF_SB-1456: remove stac extension from collections

### PR DESCRIPTION
**Pull request by hansmannj**
* on collection level, stac_extensions is now []
* sample `collection.json` of the two sample collections is corrected accordingly

### CODE REVIEW

File: app/stac_api/models.py [hunk 1]
```diff
@@ -46,8 +46,11 @@
 DEFAULT_SUMMARIES_VALUE = {"eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []}
 
 
-def get_default_stac_extensions():
-    return list(DEFAULT_STAC_EXTENSIONS.values())
+def get_default_stac_extensions(is_collection=False):
+    if is_collection: # pylint: disable=no-else-return
+        return list()
+    else:
+        return list(DEFAULT_STAC_EXTENSIONS.values())
```
Comment (ltshb) [inline app/stac_api/models.py:53]: Don't disable pylint here simply write code accordingly to it:
```
if is_collection:
    return list()
return list(DEFAULT_STAC_EXTENSIONS.values())
```

**Review by ltshb — CHANGES_REQUESTED** [review]
👍  just fixes pylint issue see comment

File: app/stac_api/models.py (hunk 1 above)
Comment (hansmannj) [reply app/stac_api/models.py]: Makes sense :-)

**Review by ltshb — APPROVED** [review]
:1st_place_medal:

**Pull request merged**